INSTALL.rst

```diff
@@ -155,3 +178,35 @@ tests::
 
    export SKIMAGE_TEST_STRICT_WARNINGS=False
    pytest --pyargs skimage
+
+3. Linux operating system installation
+--------------------------------------
+We do not recommend using the opearting system's environment of python for a few
```

Comment:
We do not recommend using the operating system's environment of python for a few